css/css-transitions/changing-while-transition-002.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html>
+<head>
+<meta charset=utf-8>
+<title>CSS Transitions Test: behavior when transition-timing-function changes while transitioning</title>
+<meta name="assert" content="Checks a change to the transition-timing-function
+property does not affect an in-flight transition">
+<link rel="help" title="3. Starting of transitions" href="https://drafts.csswg.org/css-transitions/#starting">
+
+<script src="/resources/testharness.js" type="text/javascript"></script>
+<script src="/resources/testharnessreport.js" type="text/javascript"></script>
+<script src="./support/helper.js" type="text/javascript"></script>
+
+</head>
+<body>
+<div id=log></div>
```

Comment:
Consistency, and shall I say taste, would dictate double quotes around `log`. This is true of a other tests as well.
```suggestion
<div id="log"></div>
```

Reply:
Fair enough. I've started dropping unnecessary quotes more and more. I guess my taste is getting worse :)

Reply:
This was said (mostly) in jest, this is quite a subjective topic. But I think we ought to be consistent and use quotes around attributes like in other places.